pipelines/services/types.go

```diff
@@ -0,0 +1,32 @@
+package services
+
+import "time"
+
+type PipResponse struct {
+	TotalCount int         `json:"totalCount"`
+	Pipelines  []Pipelines `json:"pipelines"`
```

Comment:
Let's add omitempty in all fields in the structs in this file
```suggestion
	TotalCount int         `json:"totalCount,omitempty"`
	Pipelines  []Pipelines `json:"pipelines,omitempty"`
```